app/frontend/components/shared/permit-applications/requirement-form.tsx

```diff
@@ -77,7 +70,12 @@ export const RequirementForm = observer(
     const [firstComponentKey, setFirstComponentKey] = useState(null)
     const [isCollapsedAll, setIsCollapsedAllState] = useState(false)
 
-    const clonedSubmissionData = useMemo(() => R.clone(submissionData), [submissionData])
+    const [unsavedSubmissionData, setUnsavedSubmissionData] = useState(R.clone(submissionData))
```

Comment:
const [unsavedSubmissionData, setUnsavedSubmissionData] = useState(() => R.clone(submissionData)), to prevent R.clone running  on each render (it doesn't change the state but the function runs any way). More info `https://react.dev/reference/react/useState#avoiding-recreating-the-initial-state`